odml/__init__.py

```diff
@@ -1,9 +1,16 @@
-import doc, section, property, value
-from odml.types import DType
+import sys
+from odml import doc
+from odml import property
+from odml import section
+from odml import value
+from odml.dtypes import DType
 
 # the original property-function is overwritten
 # so get it back!
-from __builtin__ import property as _property
+if sys.version_info < (3, 0):
+    from __builtin__ import property as _property
+else:
+    from builtins import property as _property
```

Comment:
Maybe instead of version checking directly, it would be more robust to just save the original property function before import?
```python
property_ = property
from odml import property
```

Might be more resilient to (potential) future changes in the name for the builtin module or any other weirdness.